[2021-10-18 17:19:34,363] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_2 2021-10-17T00:00:00+00:00 [queued]>
[2021-10-18 17:19:34,368] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_2 2021-10-17T00:00:00+00:00 [queued]>
[2021-10-18 17:19:34,368] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-10-18 17:19:34,368] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-10-18 17:19:34,368] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-10-18 17:19:34,386] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_2> on 2021-10-17T00:00:00+00:00
[2021-10-18 17:19:34,387] {standard_task_runner.py:52} INFO - Started process 5093 to run task
[2021-10-18 17:19:34,392] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_2', '2021-10-17T00:00:00+00:00', '--job-id', '50', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmpz2bdge2e', '--error-file', '/tmp/tmpf85cri4h']
[2021-10-18 17:19:34,393] {standard_task_runner.py:77} INFO - Job 50: Subtask task_2
[2021-10-18 17:19:34,478] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.task_2 2021-10-17T00:00:00+00:00 [running]> on host airflowvm
[2021-10-18 17:19:34,582] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_2
AIRFLOW_CTX_EXECUTION_DATE=2021-10-17T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-17T00:00:00+00:00
[2021-10-18 17:19:34,582] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-10-18 17:19:34,583] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2021-10-18 17:19:34,586] {subprocess.py:75} INFO - Output:
[2021-10-18 17:19:37,588] {subprocess.py:83} INFO - Command exited with return code 0
[2021-10-18 17:19:37,627] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_2, execution_date=20211017T000000, start_date=20211018T171934, end_date=20211018T171937
[2021-10-18 17:19:37,663] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-10-18 17:19:37,677] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-10-18 17:51:19,984] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_2 2021-10-17T00:00:00+00:00 [queued]>
[2021-10-18 17:51:20,008] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_2 2021-10-17T00:00:00+00:00 [queued]>
[2021-10-18 17:51:20,012] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-10-18 17:51:20,013] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-10-18 17:51:20,013] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-10-18 17:51:20,046] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_2> on 2021-10-17T00:00:00+00:00
[2021-10-18 17:51:20,048] {standard_task_runner.py:52} INFO - Started process 14269 to run task
[2021-10-18 17:51:20,062] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_2', '2021-10-17T00:00:00+00:00', '--job-id', '4', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmphdehs2r_', '--error-file', '/tmp/tmpgbu0cn5y']
[2021-10-18 17:51:20,065] {standard_task_runner.py:77} INFO - Job 4: Subtask task_2
[2021-10-18 17:51:20,728] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.task_2 2021-10-17T00:00:00+00:00 [running]> on host airflowvm
[2021-10-18 17:51:20,927] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_2
AIRFLOW_CTX_EXECUTION_DATE=2021-10-17T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-17T00:00:00+00:00
[2021-10-18 17:51:20,928] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-10-18 17:51:20,928] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2021-10-18 17:51:20,943] {subprocess.py:75} INFO - Output:
[2021-10-18 17:51:23,946] {subprocess.py:83} INFO - Command exited with return code 0
[2021-10-18 17:51:24,022] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_2, execution_date=20211017T000000, start_date=20211018T175119, end_date=20211018T175124
[2021-10-18 17:51:24,085] {taskinstance.py:1245} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-10-18 17:51:24,119] {local_task_job.py:151} INFO - Task exited with return code 0
